File: AudioAndVideoDevelopmentAdvanced/CrossCompiling/LAMECompileMP3/LAMEMp3Encoder/LAMEMp3Encoder/ViewController.m
//
//  ViewController.m
//  LAMEMp3Encoder
//
//  Created by 冯才凡 on 2020/7/15.
//  Copyright © 2020 冯才凡. All rights reserved.
//

#import "ViewController.h"
#import "Mp3Encoder.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [Mp3EncoderOC new].encoder;
}


@end

File: GPUImage/Projects/GPUImageDemo/GPUImageDemo/ViewController.swift
//
//  ViewController.swift
//  GPUImageDemo
//
//  Created by 冯才凡 on 2019/9/26.
//  Copyright © 2019 冯才凡. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }

    @IBAction func addAction(_ sender: Any) {
        self.present(GPUController(), animated: true) {
            
        }
    }
    
}

